.github/ISSUE_TEMPLATE/1-bug-report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: "\U0001F41B Bug report"
+about: Report a bug with the specification
+
+---
+
+<!--
+Thank you for reporting a possible bug in ECMA-262.
+
+If this is a bug with a specific implementation of ECMA-262, such as
+a JavaScript engine or a browser, please report it on their bug tracker
+instead.
+
+eshost is a tool to run the same code on many JS engines
+to compare the output.
+
+To set up eshost:
+
+1. $ npm i -g eshost-cli jsvu
+2. $ jsvu
+3. $ eshost --configure-jsvu
+
+Running eshost:
+
+Use `-x` to evaluate multiple statements:
+$ eshost -s -x "const a = 1 + 1; print(a);"
+
+Or `-e` to evaluate a single expression:
+$ eshost -s -e "1 + 1"
+
+Paste the output of eshost into the box below.
+-->
+
+**Description:** Adding 1 and 1 is two, but it should be...
+
+**eshost Output:**
+
+```js
```

Comment:
Probably remove the `js` here.

Reply:
It colors things that looks like js pretty well. I think its worth keeping, given that most of the output will be js.

Reply:
Hmm, the only JS in the output is wrapped in quotes, which would be highlighted treated as a string literal. Maybe the result value is a JS primitive as well, but that's it.